### PR TITLE
aircrack-ng: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/aircrack-ng.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# aircrack-ng
+
+> Crack WEP and WPA/WPA2 keys from handshake in captured packets.
+> Part of `aircrack-ng`.
```
Comment (kbdharun) [inline pages/common/aircrack-ng.md:4]:
```suggestion
```
Wouldn't the page title already convey that it is `aircrack-ng`?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/aircrack-ng.md (hunk 1 above)
Comment (johandegn) [reply pages/common/aircrack-ng.md:4]: Yes, and i see where the confusion might appear. aircrack-ng is both the name of the software suit this program belongs to (a collection of WIFI tools), and the name of this program itself. The existing pages for aireplay-ng, airodump-ng and airmon-ng (from this collection) all included this line. So for consistency i thought it made sense also to include it here, but let me know what you think?

Comment (kbdharun) [reply pages/common/aircrack-ng.md:4]: Yeah, that was to reference the suite, I don't think it is required here as it is the same as the program name.

Comment (blueskyson) [reply pages/common/aircrack-ng.md:4]: This is better if you really want to mention it.
```suggestion
Part of Aircrack-ng network software suite.
```

Comment (kbdharun) [reply pages/common/aircrack-ng.md:4]: Oops, you missed the `>`, I will update your suggestion.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**